### ISSUE TITLE
how to find the peretrain models

### ISSUE THREAD

**Issue by walkingmanroad** [qidiso/mobilefacenet-V2#11]
how to find the peretrain models,i cannot find in this:
https://github.com/aidlearning/AidLearning-FrameWork/tree/master/examples/facencnn
cannot find weight files?

**Comment by GokulNC** [comment]
Yeah, even I don't find the trained model that the repo author has quoted in his README.

@qidiso Can you please upload the trained model for the accuracies that you have finally quoted?
Thanks :)

**Comment by aidlearning** [comment]
I fixed the url:
https://github.com/aidlearning/AidLearning-FrameWork/tree/master/examples/facencnn

**Comment by quocnhat** [comment]
the link is dead, please give me the pretrained model and the weight file.

**Comment by GokulNC** [comment]
@quocnhat : Here's the new link:  
https://github.com/aidlearning/AidLearning-FrameWork/tree/master/src/facencnn/models

All I had to do was browse the repo patiently for a minute. :)
